Sources/Utility/Git.swift

```diff
@@ -57,7 +57,6 @@ public class Git {
         return knownVersions
     }
 
-    public class var tool: String {
-        return "git"
-    }
+    /// A shell command to run for Git. Can be either a name or a path.
+    public static var tool: String = "git"
```

Comment:
Initially Eric had almost exactly that (the override with the built-in "git" literal if there was no override), but I suggested that it would be simpler to just make the variable mutable.  How strongly do you feel that it should be an override with a fallback rather than a setting with a default value?